api/routers/site.js

```diff
@@ -9,6 +9,7 @@ router.use(csrfProtection);
 
 router.get('/site', sessionAuth, SiteController.findAllForUser);
 router.post('/site/user', sessionAuth, SiteController.addUser);
+router.delete('/site/:site_id/user/:user_id', sessionAuth, SiteController.removeUser);
```

Comment:
I think I'd like to break this up into a separate router/controller pair, along with the POST method above. I think that route could also be normalized to `/site/:site_id/user`.

That would need to be a separate refactoring PR, since parts of the front end would have to change as well.